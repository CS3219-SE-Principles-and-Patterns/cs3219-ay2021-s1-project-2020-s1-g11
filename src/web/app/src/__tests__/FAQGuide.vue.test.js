import {createLocalVue, shallowMount} from '@vue/test-utils'
import ElementUI from 'element-ui';
import VueRouter from 'vue-router';
import Vuex from 'vuex';
import FAQGuide from "./../components/userPageDetail/FAQGuide.vue";

const localVue = createLocalVue();

localVue.use(Vuex);
localVue.use(VueRouter);
localVue.use(ElementUI);

describe("FAQGuide.vue", () => {
  let store;

  beforeEach(() => {
    store = new Vuex.Store({
      state: {
        isPageLoading: false,
        userInfo: {
          isApiError: false,
        },
        dbMetaData: {
          entitiesStatus: {
            isLoading: false,
          }
        }
      }
    })
  });

  it('renders correctly', () => {
    const wrapper = shallowMount(FAQGuide, {store, localVue})
    expect(wrapper.element).toMatchSnapshot()
  });
});