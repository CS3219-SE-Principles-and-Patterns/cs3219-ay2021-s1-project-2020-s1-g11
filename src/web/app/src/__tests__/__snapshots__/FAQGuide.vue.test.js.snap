// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`FAQGuide.vue renders correctly 1`] = `
<el-collapse-stub
  value=""
>
  <el-collapse-item-stub
    name="1"
    title="How to use our user guide?"
  >
    
    We have provided 4 different panels for our user guide:
    
    <ol>
      <li>
         Overview
      </li>
       
      <li>
         Features - To describe the features that our system can provide 
      </li>
       
      <li>
         Instructions - To provide a detailed instructions on how to use ChairVise features 
      </li>
       
      <li>
         FAQ - To list out commonly asked questions by user 
      </li>
    </ol>
    
    To use our user guide, click on any of the tabs above to begin exploring.
  
  </el-collapse-item-stub>
   
  <el-collapse-item-stub
    name="2"
    title="My chart seems to overlapped"
  >
    <p>
       Scroll all the way to the top and click "PDF" / "Powerpoint" again. 
    </p>
  </el-collapse-item-stub>
   
  <el-collapse-item-stub
    name="3"
    title="Cannot upload Review Record with no header for EasyChair"
  >
    <p>
       Use the with header option. 
    </p>
  </el-collapse-item-stub>
</el-collapse-stub>
`;
